src/objects/_width-container.scss

```diff
@@ -5,15 +5,37 @@
   // On mobile, add half width gutters
   margin: 0 $govuk-gutter-half;
 
+  // Add safe area (e.g. iPhone X, Google Pixel 3)
+  @supports (margin: unquote("max(calc(0px))")) {
+    // Use safe area or mobile margin, whichever is larger
+    // Escaped due to Sass max() vs. CSS native max()
+    margin-right: unquote("max(#{$govuk-gutter-half}, calc(#{$govuk-gutter-half} + env(safe-area-inset-right)))");
+    margin-left: unquote("max(#{$govuk-gutter-half}, calc(#{$govuk-gutter-half} + env(safe-area-inset-left)))");
+  }
+
   // On tablet, add full width gutters
   @include govuk-media-query($from: tablet) {
     margin: 0 $govuk-gutter;
+
+    // Add safe area (e.g. iPhone X, Google Pixel 3)
+    @supports (margin: unquote("max(calc(0px))")) {
+      // Use safe area or tablet margin, whichever is larger
+      // Escaped due to Sass max() vs. CSS native max()
+      margin-right: unquote("max(#{$govuk-gutter}, calc(#{$govuk-gutter-half} + env(safe-area-inset-right)))");
```

Comment:
Is this intentionally using `$govuk-gutter-half` rather than `$govuk-gutter` within the calc?

Reply:
It is yeah. It's equals "notch safe area plus a little bit more" for aesthetics.

Adding the full gutter was too much.

Reply:
Just to be clear, I'm asking if it's intentional that we're using _both_ `$govuk-gutter` and `$govuk-gutter-half` on this line – on the two different sides of the `max` calculation.

Reply:
I think maybe clearer comments or new private variables may make this clearer...

Reply:
I've changed all the padding/margin overrides to use variables now:

```css
margin-right: unquote("max(#{$govuk-gutter}, #{$gutter-safe-area-right})");
margin-left: unquote("max(#{$govuk-gutter}, #{$gutter-safe-area-left})");
```

This conversation is on an outdated diff so open another comment if you've got some more feedback? I'm happy to fit in with both of you @nickcolley @36degrees.